tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -73,7 +74,11 @@ def provision(self):
 
         archive = FunctionPackage(self.policy.data)
         archive.build()
-        archive.publish(parameters['name']['value'])
+
+        if archive.status(parameters['name']['value']):
```

Comment:
Let's move `parameters['name']['value']` to a local variable.

Reply:
Oh it was one, just wasn't being used :)